src/client/datascience/jupyter/liveshare/hostJupyterNotebook.ts

```diff
@@ -188,233 +89,23 @@ export class HostJupyterNotebook
         try {
             return super.interruptKernel(timeoutMs);
         } catch (exc) {
-            this.postException(exc, []);
             throw exc;
         }
     }
 
-    private makeRequest(
-        code: string,
-        file: string,
-        line: number,
-        id: string,
-        silent: boolean | undefined,
-        responseQueues: ResponseQueue[]
-    ): Promise<ICell[]> {
-        // Create a deferred that we'll fire when we're done
-        const deferred = createDeferred<ICell[]>();
-
-        // Attempt to evaluate this cell in the jupyter notebook
-        const observable = this.makeObservableRequest(code, file, line, id, silent, responseQueues);
-        let output: ICell[];
-
-        observable.subscribe(
-            (cells: ICell[]) => {
-                output = cells;
-            },
-            (error) => {
-                deferred.reject(error);
-            },
-            () => {
-                deferred.resolve(output);
-            }
-        );
-
-        // Wait for the execution to finish
-        return deferred.promise;
-    }
-
     private makeObservableRequest(
         code: string,
         file: string,
         line: number,
         id: string,
-        silent: boolean | undefined,
-        responseQueues: ResponseQueue[]
+        silent: boolean | undefined
     ): Observable<ICell[]> {
         try {
             this.requestLog.set(id, Date.now());
```

Comment:
Shouldn't need the requestLog anymore. It was for tracking requests to make sure sent to both sides.